nautobot/core/templates/components/layout/one_over_two.html

```diff
@@ -0,0 +1,19 @@
+{% load ui_framework %}
+{% load plugins %}
+
+<div class="row">
+    <div class="col-md-12">
+        {% render_components full_width_panels %}
+        {% if include_plugin_content %}{% plugin_full_width_page object %}{% endif %}
```

Comment:
We should open an issue to translate `TemplateExtension` instances into `Panel` subclasses, and allow them to define appropriate `weight` values so as to be interleavable with baseline content, instead of needing a separate set of APIs specifically for App extensions.

Reply:
#6276